.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: scala
 scala:
-   - 2.10.5
-   - 2.11.7
+   - 2.10.6
+   - 2.11.8
```

Comment:
As I described in my previous comments scala 2.10 has weird bug with Streams and tranquility fails with OOM. I think it's better to merge #221 first and then merge this PR.